webrtc-stats.html

```diff
@@ -1258,8 +1267,8 @@
               </dt>
               <dd>
                 <p>
-                  Represents the RTT computed by the STUN connectivity checks and measured
-                  in seconds [[!STUN-PATH-CHAR]].
+                  Represents the RTT measured in seconds, computed from both STUN
+                  connectivity checks [[!STUN-PATH-CHAR]] and consent responses [[!RFC7675]].
```

Comment:
@aboba. Is this the average RTT for the ICE candidate pair across both mechanisms? 

In RTP we report the last RTT measurement.